sql/core/src/main/scala/org/apache/spark/sql/streaming/StreamingQueryManager.scala

```diff
@@ -237,7 +237,7 @@ class StreamingQueryManager private[sql] (sparkSession: SparkSession) extends Lo
           "is not supported in streaming DataFrames/Datasets and will be disabled.")
     }
 
-    new StreamingQueryWrapper(new StreamExecution(
+    new StreamingQueryWrapper(new MicroBatchExecution(
```

Comment:
shall we make it class name configurable?

Reply:
I mean, how we switch between ContinuousExecution and MicroBatchExecution?

Reply:
My current thinking is to have it be a new trigger type. It can't really be a config, because continuous processing (at least in the initial implementation) won't support all operators.